docs/xmlprofiles.rst

```diff
@@ -1384,8 +1437,28 @@ See :ref:`realtime-allocations` for detailed information on how to tune allocati
      - ``UInt32``
      - 1
 
+.. _MessageMaxSize:
+
+Submessage Size Limit
+^^^^^^^^^^^^^^^^^^^^^
+
+While some submessages have a fixed size (for example, SequenceNumber), others have a variable size depending
+on the data they contain.
+
+Processing a submessage requires having a memory chunk large enough to contain a copy of its data.
+That's easy with fixed variable submessages, as size is known and memory can be allocated beforehand.
+For variable size submessages we can use two strategies:
+
+    - Set a maximum size for the data container, that will be allocated beforehand during the setup of
+      the participant. This avoids dynamic allocations during message communication, but any submessage
+      with a larger payload than the defined maximum will not fit in, and will be discarded.
+    - Do not set any maximum and allocate the required memory dynamically upon submessage arrival,
+      according to the size declared on the header. This allows any size of submessages at the cost of
+      dynamic allocations during message decoding.
```

Comment:
```suggestion
While some submessages have a fixed size (for example, SequenceNumber), others have a variable size depending
on the data they contain. Processing a submessage requires having a memory chunk large enough to contain a copy of the
submessage data. That is easy to handle when dealing with fixed variable submessages, as size is known and memory can be
allocated beforehand. For variable size submessages on the other hand, two different strategies can be used:

    - Set a maximum size for the data container, which will be allocated beforehand during the participant's setup. This
      avoids dynamic allocations during message communication. However, any submessages with a larger payload than the
      defined maximum will not fit in, and will therefore be discarded.
    - Do not set any maximum size for the data container, and instead allocate the required memory dynamically upon
      submessage arrival (according to the size declared on the submessage header). This allows for any size of
      submessages, at the cost of dynamic allocations during message decoding.
```

Reply:
We have the policy of breaking line when a sentence ends. When the documentation is built, Paragraphs are defined by blank lines

Reply:
Suggestion updated